src/module.rs

```diff
@@ -118,6 +125,43 @@ impl<'a> fmt::Display for Module<'a> {
     }
 }
 
+fn ansi_strings_modified(ansi_strings: Vec<ANSIString>, shell: String) -> Vec<ANSIString> {
+    ansi_strings
+        .iter()
+        .map(|ansi| {
+            let mut escaped = false;
+            let final_string: String = ansi
+                .to_string()
+                .chars()
+                .map(|x| match x {
+                    '\u{1b}' => {
+                        escaped = true;
+                        match shell.as_str() {
+                            "bash" => String::from("\u{5c}\u{5b}\u{1b}"), // => \[ESC
```

Comment:
My initial thought was to have the `ansi_strings` function pass the escape sequences into this function, but after some thought, I think your method is much more extensible 👍 .